readyness/module.go

```diff
@@ -0,0 +1,14 @@
+package readyness
+
+import (
+	"go.uber.org/fx"
+)
+
+var Module = fx.Options(
+	fx.Provide(
+		NewServiceReadyness,
```

Comment:
Why is this exported?

Reply:
because i just ddid copy and paste. Will also make the equivalent in the `health` package unexported 👍